Project-18-Food-order-app/src/hooks/useInput.js

```diff
@@ -0,0 +1,28 @@
+import { useState } from "react";
+
+function useInput(defaultValue, validationFn) {
+  const [enteredValues, setEnteredValues] = useState(defaultValue);
+
+  const [didEdit, setDidEdit] = useState(false);
+
+  const isValueValid = validationFn(enteredValues);
```

Comment:
@sknepali Consider using more descriptive variable names to improve readability and understandability of the code.